src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidNoArgumentSuperConstructorCallCheckExamplesTest.java

```diff
@@ -35,9 +33,9 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
+            "20:5: " + getCheckMessage(AvoidNoArgumentSuperConstructorCallCheck.MSG_CTOR),
```

Comment:
Not that this is wrong but By convention, we usually import the key first, and then use it in a violation string.
Please see #14430 for reference